docs/reference/certificaterequests.rst

```diff
@@ -0,0 +1,77 @@
+     isCA: false
+     duraton: 90d
+     issuerRef:
+       name: ca-issuer
+       # We can reference ClusterIssuers by changing the kind here.
+       # The default value is Issuer (i.e. a locally namespaced Issuer)
+       kind: Issuer
+       group: certmanager.k8s.io
+
+This CertificateRequest will make cert-manager attempt to make the Issuer
+``letsencrypt-prod`` in the default issuer pool ``certmanager.k8s.io``, return a
+certificate based upon the certificate signing request. Other groups can be
+specified inside the ``issuerRef`` which will change the targeted issuers to other
+external, third party issuers you may have installed.
+
+The resource also exposes the option for stating the certificate as CA and
+requested validity duration.
+
+A successful issuance of the certificate signing request will cause an update to
+the resource, setting the status with the signed certificate, the CA of the
+certificate (if available), and setting the `Ready` condition to `True`.
+
+Whether issuance of the controller was successful or not, a retry of the
+issuance will _not_ happen. It is the responsibility of some other controller to
+manage the logic and life cycle of CertificateRequets.
+
+----------
+Conditions
+----------
+
+CertificateRequests have a set of strongly defined conditions that should be
+used and relied upon by controllers or services to make decisions on what
+actions to take next on the resource. Each condition consists of the pair
+`Ready` - a boolean value, and `Reason` - a string. The set of values and
+meanings are as follows:
+
++---------+-----------------+---------------------------------------------------------------------------------------------------------------------------+
+| *Ready* | *Reason*        | Condition Meaning                                                                                                         |
++=========+=================+===========================================================================================================================+
+| False   | CertPending     | A valid Issuer has been found and is currently in the process of issuing a certificate based on the Spec of the resource. |
```

Comment:
Can we remove the `Cert` prefix here? (and below)

Reply:
Going to make a separate PR for this since it's code change